### DISCUSSION TITLE
[x] Developing connections and communication throughout your institution

### DISCUSSION THREAD

**Discussion by betsybookwyrm** [RSEAA/RSEAA.github.io#58] [Discussion topics for #RSEAA2023]
There are pockets of software everywhere in universities. Larger pockets such as eResearch, research infrastructure, and computationally-focused groups. Individual researchers, students, RAs, or contractors. Professional staff developing software situated in a research group.

How do we find each other? How do we make connections and share knowledge? How can the larger groups help those who are more isolated, and how can those who are more independent help bring new ideas to the larger groups?

**Comment by betsybookwyrm** [comment]
As a non-RSE (or only partially RSE) example, at my university (I work in research infrastructure) we've recently started a Research Support Services Community of Practice. We have central groups from the library to the graduate degree support teams to eResearch and more - but we know there are more research support groups at our university, probably situated in faculties, we just don't know exactly where. We also have no idea how to find individual research support staff situated in schools or research groups. And once we find them, should we create a mailing list? A website? A slack/Teams channel? How can we best share information? We're only just starting to discuss these things amongst ourselves.

I'd love to hear what challenges and successes others have had finding RSEs across your institutions and developing communication and skills/knowledge sharing channels.

**Comment by betsybookwyrm** [comment]
(Yes this is prompted by comments from Kiowa's keynote!)

**Comment by alfski** [comment]
At WSU we started running Research Hacky Hours drop-in face-to-face and then Zoom sessions with expertise from Library, Research Services and ITDS.
Unfortunately, they quickly moved to general research-support Q&A, eg. How do I create a bibliography? Where can I install SAS?
So I don't have a good solutions for RSE's!

**Reply by betsybookwyrm** [reply]
this is very similar to our hacky hour experience!

**Comment by Aleem2** [comment]
We are looking to have a hacky session or a support session for one of our community. However where do we start?

**Reply by nrmay** [reply]
You could have a look at the Hacky Hour Handbook
https://github.com/amandamiotto/HackyHourHandbook

**Reply by richarddushime** [reply]
i m from reading through and found it amazing

**Comment by kiowa-scott-hurley** [comment]
Picking a platform to connect on is such a hurdle! 

I've found having some sort of regular video-call meeting is a good way to stay in touch with people, even if the other connection-platform-of-choice isn't ideal. Of course, that means you need a person to do the work of organizing these regular catch-ups - it's a hard job in my experience, and not one that's well-resourced.

**Comment by chesadi** [comment]
Multidisciplinary projects and activities may help to develop the connections and communication.

**Reply by robmoss** [reply]
This is a great reason to have a "hub-and-spoke" model where RSEs can be grouped together, and collaborate with different groups on an as-needed basis.

**Comment by alfski** [comment]
paper on Princeton Research Software Engineering Group Model https://ieeexplore.ieee.org/document/10090857

**Comment by betsybookwyrm** [comment]
My notes:

- at Princeton they've set up a central cell of research software engineers and they lend them out to research projects at 50% cost. They've grown rapidly - research projects try to poach the staff - they preserve 15% of the time of staff locally for staff to come back to the central group and stay in touch - https://ieeexplore.ieee.org/document/10090857
- Aleem @ ARDC - they've created communities of interest
	- created a terms of reference to define the community
	- regular meetings
- Kiowa - HPC community
	- HPC slack space is very active, 
	- they also have regular unconferencey calls
	- scheduling the meetups is easy, but the coming up with topics etc is really hard volunteer work that goes unrecognised. Longevity is hard.
- Johan
	- My experience has been that community managers are needed to maintain things
	- Leaving space for the community to focus on topics of interest, not on the management aspects of communities
	- Combined with community led use cases; Breaking large tasks into tasks that can be managed; Finding community champions
- Alf
	- beware of big personalities that will then leave a big hole when they leave
	- rotating chairs is a good way to make the group less dependent on one person
- W.A. Indika
	- finding the right mix of people cross-domain is a challenge
- Recognition - community development usually doesn't fit into KPIs
- Johan - Biocommons
	- Biocommons has actual community managers! Community management became part of Johan's job description as they realised how important it was
	- Finding the voice of a community is the challenge
	- quarterly meetings for each subcommunity
	- biocommons made this happen because community is central to what the biocommons is - their role is to look across their community and find common needs
- How do you convince people that a community manager for RSEs is worth the money?
	- they can support underrepresented groups
	- they can find common needs to fill
	- ???
- UK, US, and AuNZ RSE groups all have different models, we can learn from other groups
- lack of job security is a huge impediment - people can't take on community management work when they have to focus on getting their contract renewed
- Action:
	- RSEAuNZ to collect success stories and disaster stories

**Reply by kozo2** [reply]
What kind of activities would be necessary to create such a mechanism in our own organization?

**Comment by robmoss** [comment]
The University of Melbourne has a somewhat-similar group: [Melbourne Data Analytics Platform](https://mdap.unimelb.edu.au/)

**Reply by rowlandm** [reply]
[ADACS](https://adacs.org.au/) is another group like this.

[RCP at WEHI ](https://wehi-researchcomputing.github.io/)is doing something similar

**Reply by kozo2** [reply]
Why does Australia have Research / Data Commons but Japan does not?
How did such an organization come to be in Australia?

**Reply by rowlandm** [reply]
Most of them naturally developed (I think) and some followed from the UK's RSE setup over the last 10 years....

**Comment by rowlandm** [comment]
Burnout in community and volunteerism is a big thing ...

**Reply by kiowa-scott-hurley** [reply]
- which is leading to people reducing the amount of "extra-curriculars" they participate in, and prioritizing their efforts

**Reply by kiowa-scott-hurley** [reply]
- prioritizing things that give the most energy back
- prioritizing things that are recognized formally in work hours and that will support work stability

**Reply by betsybookwyrm** [reply]
Another thing I didn't mention about the community management + burnout issue - in our area this sort of stuff is treated as glue work. Glue work is disproportionally gendered. The negative impact of burning out or missing out on career recognition of community management is disproportionally borne by women. This is an issue that is being discussed in some areas of the tech industry, but I don't think the discussion is widespread and I don't know if the term is used in an academic environment context. See https://medium.com/@granellacamila/a-few-months-ago-i-discovered-the-term-glue-work-8a003dbe7173 for a starting place for reading up on the concept of glue work. The effect in our area is likely masked by the gender imbalance in our field. Also worth noting is that the burden of volunteering is disproportionally borne by those who are part of multiple underrepresented groups.

**Reply by kiowa-scott-hurley** [reply]
I've seen a lot about the Diversity Tax in academia, to your last point (https://www.nature.com/articles/d41586-020-01920-6) ...but haven't heard the term glue work before!

**Reply by betsybookwyrm** [reply]
I haven't encountered the term diversity tax before! I'll read that article, thanks

**Reply by betsybookwyrm** [reply]
oh gosh yes I have 100% observed that diversity tax phenomenon. That's a very good name for it!

**Comment by rowlandm** [comment]
My experience has been that community managers are needed to maintain things
Leaving space for the community to focus on topics of interest, not on the management aspects of communities
Combined with community led use cases
Breaking large tasks into tasks that can be managed
Finding community champions

**Reply by supernord** [reply]
thanks for adding this @rowlandm 
this is largely in the context of national communities that I engage with

**Comment by kiowa-scott-hurley** [comment]
- sustainability of groups is tricky
- rotating the "chair" or the roles can help with this
- identify the "big personalities" or the volunteers and make sure they have supports so they don't burn out, and that others can take over the role if needed

**Comment by rowlandm** [comment]
Does the precariousness of work in RSE mean we are less likely to volunteer?

**Comment by rowlandm** [comment]
community managers/builders are definitely not recognised enough… probably everywhere.

**Reply by rowlandm** [reply]
What about long term funding?

**Comment by rowlandm** [comment]
What models are available for RSEs within a university context when there is only mainly non-continuing funding (ie. grants)?

**Comment by kiowa-scott-hurley** [comment]
How can we weaponise competition in the market to encourage better working conditions of RSEs? (AKA USyd offering remote work!) 

How can we weaponise stories about how hard it is to replace RSEs / bring newstarters up to speed to start working on the job security problem?
